### PR TITLE
Added docstrings examples for clique.py

### PR DESCRIPTION
**Pull request by Qudirah**
#6575 @MridulS have a look :)

### CODE REVIEW

File: networkx/algorithms/approximation/clique.py [hunk 1]
```diff
@@ -40,6 +40,14 @@ def maximum_independent_set(G):
     iset : Set
         The apx-maximum independent set
 
+    Examples
+    --------
+    >>> import networkx as nx
```
Comment (rossbar) [inline networkx/algorithms/approximation/clique.py:45]:
```suggestion
```

The `import networkx as nx` is automatically included for all docstring examples and should not be explicitly added.

File: networkx/algorithms/approximation/clique.py [hunk 2]
```diff
@@ -40,6 +40,14 @@ def maximum_independent_set(G):
...
+    >>> from networkx.algorithms.approximation.clique import maximum_independent_set
+    >>> G = nx.path_graph(10)
+    >>> maximum_independent_set(G)
```
Comment (rossbar) [inline networkx/algorithms/approximation/clique.py:48]: Similar to the other PRs you've opened related to import styles, we want to encourage users to access the functions from the proper namespace:

```suggestion
    >>> G = nx.path_graph(10)
    >>> nx.approximation.maximum_independent_set(maximum_independent_set(G))
```

**Review by rossbar — COMMENTED** [review]
I've left two explicit comments but note that they apply to all the proposed examples!

**Comment by Qudirah** [comment]
Thank you everyone @dschult @MridulS @rossbar

**Review by dschult — APPROVED** [review]

**Pull request merged**